### PR TITLE
Fix/change naming

### PR DESCRIPTION
**Pull request by Eronavrp**
Changed naming of :
- Project to Team 
- ProjectJob to Project.

### CODE REVIEW

**Review by FabianNowak — COMMENTED** [review]
Is the change to Frontend PopUp.jsx intended?

File: Backend/src/API/REST/routes/team/teamRouter.ts [hunk 1]
```diff
@@ -33,67 +33,67 @@ const create_project = () : Router => {
...
 
             const user = await UserService.getUserById(userId);
             if (! user )
                 return res.status(400).send("User to be invited does not exist.");
 
-            const project = await ProjectService.getProjectById(projectId);
-            if (! project)
+            const team = await TeamService.getTeamById(teamId);
+            if (! team)
                 return res.status(400).send("Project does not exist.");
```
Comment (FabianNowak) [inline Backend/src/API/REST/routes/team/teamRouter.ts:73]: Should this be "Team does not exist"?

Comment (FabianNowak) [inline Backend/src/API/REST/routes/team/teamRouter.ts:73]:
```suggestion
                return res.status(400).send("Team does not exist.");
```

File: Backend/src/database/models/project.ts [hunk 2]
```diff
@@ -1,63 +1,51 @@
...
+    owner: Schema.Types.ObjectId;
+
+    // file
+    uploadId: string;
+    location: string;
+    fileName: string;
+    fileSize: number; // (of bytes)
+    uploadDate: Date;
+
+    // team
```
Comment (FabianNowak) [inline Backend/src/database/models/project.ts:21]:
```suggestion
    // project
```
Still saves information about the project status

File: Backend/src/database/models/project.ts [hunk 3]
```diff
@@ -1,63 +1,51 @@
...
+    location: {type: String, require: false},
+    fileSize: {type: Schema.Types.Number, require: false, default: -1},
+    uploadDate: {type: Schema.Types.Date, require: true},
 
-    invitedMemberIds: [{
-        type: Schema.Types.ObjectId, 
-        ref: 'User',
-        require: false
-    }],
+    // team
```
Comment (FabianNowak) [inline Backend/src/database/models/project.ts:39]:
```suggestion
    // project
```
Still stores information about the project status

**Review by FabianNowak — COMMENTED** [review]
On multiple occasions there is "team job" instead of "project" (most of it in comments)

**Comment by Eronavrp** [comment]
> Is the change to Frontend PopUp.jsx intended?

No it is not. It should be reversed to its previous state.

**Review by Pellumbengineer — APPROVED** [review]
LGTM

**Pull request merged**